operator/main.go

```diff
@@ -197,6 +205,14 @@ func defineFlagVar() *FlagVar {
 	flag.BoolVar(&flagVar.enableModuleCatalog, "enable-module-catalog", true,
 		"Enabling the Module Catalog Synchronization for Introspection of "+
 			"available Modules based on ModuleTemplates.")
+	flag.BoolVar(&flagVar.enableKcpWatcher, "enable-kcp-watcher", true,
+		"Enabling KCP Watcher to reconcile Watcher CRs created by KCP run operators")
+	flag.StringVar(&flagVar.skrWatcherPath, "skr-watcher-path", "./internal/skr/chart/skr-webhook",
+		"The path to the skr watcher chart.")
+	flag.StringVar(&flagVar.vsName, "virtual-svc-name", "kcp-events",
```

Comment:
Please change to dynamically create this VirtualService, otherwise we need to predefined a `VirtualService` and provide dummy Spec (it will failed the validation webhook in gardener cluster if not provide), more probmatic is keeping the name and namespace consistent between these two flags and the manifest. Agreed this solution with @jakobmoellersap, @khlifi411